Automattic/woocommerce-payments
# fix: size of payment methods card

Related to https://github.com/Automattic/woocommerce-payments/pull/2372

#### Changes proposed in this Pull Request









I noticed a few inconsistencies with the designs on the payment methods section, especially when multiple payment methods are added.
I also ensured that the list of payment methods is always visible even when UPE is disabled, per design.

Before - missing heading, spacing/alignment on the dropdown/unwanted spacing around payment methods:
![Markup 2021-07-14 at 11 58 15](https://user-images.githubusercontent.com/273592/125662436-729d3b8b-9bd0-4a74-9681-9f12c0df9471.png)


After:
![Screen Shot 2021-07-14 at 11 55 27 AM](https://user-images.githubusercontent.com/273592/125662211-9e322d97-0467-49c7-9a9b-c0b1c3bc0713.png)


#### Testing instructions





- Ensure you have "Enable UPE settings changes" enabled
- Ensure you have "Enable UPE checkout" disabled
- Go to http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=woocommerce_payments
- Section shows the CC payment method
![Screen Shot 2021-07-14 at 11 54 04 AM](https://user-images.githubusercontent.com/273592/125661783-1cd2cb3c-1d17-4ae4-8c61-572da9ad36f4.png)
- Click "Enable UPE" (right now it's just making a POST request, but we should redirect the user to the onboarding - I didn't want to make too many changes yet)
- Refresh the page (this is just a trick that won't be needed once we release to make the "add a payment method" button appear)
- Add a payment method
- UI is fixed
- Disable UPE
- Refresh the page
- Additional payment method is no longer present


-------------------
